Dockerfile

```diff
@@ -24,9 +22,11 @@ RUN apk add --update \
     rm /usr/local/bin/gosu.asc && \
     chmod +x /usr/local/bin/gosu && \
     mkdir /opt && \
-    curl -SL "https://github.com/SickGear/SickGear/archive/release_${SICKGEAR_VERSION}.tar.gz" | \
+    TAG_NAME=$(curl -s https://api.github.com/repos/SickGear/SickGear/releases | \
+      python -c "import sys, json; print json.load(sys.stdin)[0]['tag_name']") && \
```

Comment:
I just checked, and `json` is a standard Python library on 2.7. Since this is run in the actual docker container we then install SickGear in, and SickGear depends on Python, this dependency is already met.